Storage/src/Connection/ServiceDefinition/storage-v1.json

```diff
@@ -440,12 +487,9 @@
        }
       }
      },
-     "zoneAffinity": {
```

Comment:
Before we can merge this PR, we need to understand why this was removed

Reply:
This feature seems to have been removed from production because there was no real usage.